CHANGES.rst

```diff
@@ -3,8 +3,8 @@ Changelog
 
 There's a frood who really knows where his towel is.
 
-2.13b4 (unreleased)
-^^^^^^^^^^^^^^^^^^^
+3.0b1 (unreleased)
```

Comment:
@idgserpro done! do you mind if I remove now upgrade steps older than 2 years?

Reply:
Since it's a 3.x release, I see no problem with that. But you may need to create a 2.x branch and be careful about upgradeSteps numbering like we had in a (portuguese) discussion in https://github.com/plonegovbr/brasil.gov.portal/pull/471#pullrequestreview-119099167

That's why I prefer not to remove old upgradeSteps. It gives that feeling of removing and "cleaning" the package but creates another problems.

I believe our both approaches aren't wrong, it's just a matter of preference of which problems you want to live with.